File: examples/advanced-crud/src/main/java/org/example/advancedcrud/app/repository/GenreRepository.java
package org.example.advancedcrud.app.repository;

import org.example.advancedcrud.app.entity.Genre;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
import org.springframework.stereotype.Repository;

@Repository
public interface GenreRepository extends JpaRepository<Genre, Long>, JpaSpecificationExecutor<Genre> {
}

File: examples/basic-crud-kotlin/src/main/kotlin/org/example/basiccrudkotlin/app/repository/GenreRepository.kt
package org.example.basiccrudkotlin.app.repository;

import org.example.basiccrudkotlin.app.entity.Genre
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.jpa.repository.JpaSpecificationExecutor
import org.springframework.data.rest.core.annotation.RepositoryRestResource

@RepositoryRestResource(path = "genres")
interface GenreRepository : JpaRepository<Genre, Int>, JpaSpecificationExecutor<Genre> {
}